main.go

```diff
@@ -27,17 +28,16 @@ func createVehicleHandler(actorSystem goakt.ActorSystem) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		vid := r.URL.Query().Get("id")
 		logger := actorSystem.Logger()
-		pid, err := actorSystem.LocalActor(vid)
+
+		command := &pb.GetPosition{}
+		res, err := goakt.NoSender.SendSync(r.Context(), vid, command, time.Minute)
```

Comment:
@Tochemey what is the right way to make `Sendsync()` call here? The sender is the webserver itself, not another actor.